Codeinwp/neve
# Fix Post Type assignment in Infinit scroll

### Summary


The post type was overridden with the value 'post' even if the args had its value. Now, it will act as the default value.

### Will affect visual aspect of the product

NO

### Screenshots 

#### Before

![image](https://github.com/Codeinwp/neve/assets/17597852/985ed61b-7bfb-4873-b21b-47c889a71378)


#### After

![image](https://github.com/Codeinwp/neve/assets/17597852/0f2b5ac1-ef49-47ec-9529-216f1d2ab6c5)


### Test instructions


- Create a CPT and enable the archive for it
- Create some posts for that type so the Infinite scroll can be triggered (You can also decrease the posts per page number in Dashboard > Settings > Reading)
- Go to the CPT archive and check which posts are rendered when the infinite scroll is triggered

## Check before Pull Request is ready:



Closes #.
